web/src/main/java/no/nav/modiapersonoversikt/service/pdl/PdlOppslagServiceImpl.kt

```diff
@@ -124,22 +125,16 @@ open class PdlOppslagServiceImpl constructor(
     }
 
     private val userTokenAuthorizationHeaders: HeadersBuilder = {
-        when (val azureToken = AuthContextUtils.azureAdUserToken()) {
-            null -> {
-                header(NAV_CONSUMER_TOKEN_HEADER, AUTH_METHOD_BEARER + AUTH_SEPERATOR + stsService.systemUserToken)
-                header(AUTHORIZATION, AUTH_METHOD_BEARER + AUTH_SEPERATOR + AuthContextUtils.requireToken())
-            }
-            else -> {
-                header(AUTHORIZATION, AUTH_METHOD_BEARER + AUTH_SEPERATOR + oboTokenClient.exchangeOnBehalfOfToken(azureToken))
-            }
-        }
-
+        val azureToken = AuthContextUtils.requireToken()
+        header(AUTHORIZATION, AUTH_METHOD_BEARER + AUTH_SEPERATOR + oboTokenClient.exchangeOnBehalfOfToken(azureToken))
```

Comment:
Nav-callId headeren skal bli automatisk lagt på av `LogginGraphqlClient` 🤔 Så burde egentlig ikke være behov for å legge til den?